next/components/social-share.tsx

```diff
@@ -0,0 +1,61 @@
+import { useTranslation } from "next-i18next";
+import { useEffect, useState } from "react";
+
+import FacebookIcon from "../styles/icons/facebook.svg";
+import LinkedInIcon from "../styles/icons/linkedin.svg";
+import TwitterIcon from "../styles/icons/twitter.svg";
+
+export function SocialShare() {
+  const [pageUrl, setPageUrl] = useState<string>("");
+  const { t } = useTranslation();
+
+  useEffect(() => {
+    const currentUrl = window.location.href;
+    setPageUrl(currentUrl);
+  }, [pageUrl]);
+
+  const data = [
+    {
+      id: 1,
+      location: "Facebook",
+      url: `https://www.facebook.com/sharer/sharer.php?u=${encodeURIComponent(
+        pageUrl
+      )}`,
```

Comment:
As a small cleanup, we could call `encodeURIComponent` just once when we calculate it?

Reply:
Sorted.